ale/transformation.py

```diff
@@ -115,35 +115,45 @@ def from_spice(cls, *args, sensor_frame, target_frame, center_ephemeris_time, ep
         for s, d in time_dependent_frames:
             quats = np.zeros((len(times), 4))
             avs = np.zeros((len(times), 3))
-            for j, time in enumerate(times):
-                state_matrix = spice.sxform(spice.frmnam(s), spice.frmnam(d), time)
-                rotation_matrix, avs[j] = spice.xf2rav(state_matrix)
-                quat_from_rotation = spice.m2q(rotation_matrix)
-                quats[j,:3] = quat_from_rotation[1:]
-                quats[j,3] = quat_from_rotation[0]
-
-            rotation = TimeDependentRotation(quats, times, s, d, av=avs)
-            frame_chain.add_edge(rotation=rotation)
+            try:
+                for j, time in enumerate(times):
+                    state_matrix = spice.sxform(spice.frmnam(s), spice.frmnam(d), time)
+                    rotation_matrix, avs[j] = spice.xf2rav(state_matrix)
+                    quat_from_rotation = spice.m2q(rotation_matrix)
+                    quats[j,:3] = quat_from_rotation[1:]
+                    quats[j,3] = quat_from_rotation[0]
+
+                rotation = TimeDependentRotation(quats, times, s, d, av=avs)
+                frame_chain.add_edge(rotation=rotation)
+            except Exception as e:
+                pass
 
         for s, d in constant_frames:
-            quats = np.zeros(4)
-            rotation_matrix = spice.pxform(spice.frmnam(s), spice.frmnam(d), times[0])
-            quat_from_rotation = spice.m2q(rotation_matrix)
-            quats[:3] = quat_from_rotation[1:]
-            quats[3] = quat_from_rotation[0]
+            try:
+                quats = np.zeros(4)
+                rotation_matrix = spice.pxform(spice.frmnam(s), spice.frmnam(d), times[0])
+                quat_from_rotation = spice.m2q(rotation_matrix)
+                quats[:3] = quat_from_rotation[1:]
+                quats[3] = quat_from_rotation[0]
 
-            rotation = ConstantRotation(quats, s, d)
+                rotation = ConstantRotation(quats, s, d)
 
-            frame_chain.add_edge(rotation=rotation)
+                frame_chain.add_edge(rotation=rotation)
+            except Exception as e:
+                pass
```

Comment:
Why are these in try except pass blocks? This could cause a ton of silent failures.

Reply:
This is a problem with how the frame lists are being created from the frame trace. If you are trying to use Nadir pointing the frame chain still tries to calculate a rotation from the sensor to j2000 which can't be done since the sensor frame is not loaded into the kernels. 

A better way to handle this might be to modify the frame trace to return empty lists if nadir is set to true.